packages/ui/src/molecules/SlideOver/stories/SlideOver.stories.tsx

```diff
@@ -0,0 +1,39 @@
+import React, { useState } from 'react';
+
+import Component from '..';
+import { Button } from '../../../';
+import mdx from './SlideOver.mdx';
+
+import type { Story, Meta } from '@storybook/react'
+
+const DefaultTemplate: Story = () => {
+  const [isOpen, setIsOpen] = useState(false)
+  const handleClose = () => setIsOpen(false)
+
+  return (
+    <>
+      <Button onClick={() => setIsOpen(!isOpen)}>Open Slide Over</Button>
+
+      <Component
+        size="partial"
+        direction="rightSide"
+        fade="in"
+        isOpen={isOpen}
+        onDismiss={handleClose}
+      >
+        <Button onClick={handleClose}>Close Slide Over</Button>
+      </Component>
+    </>
+  )
+}
+
+export const Default = DefaultTemplate.bind({})
```

Comment:
Are we going to list another example besides `Default` version?
<img width="497" alt="image" src="https://user-images.githubusercontent.com/3356699/194303729-97b0f0f3-80cc-4cb2-abb4-3c4b9807f5df.png">